assets/js/admin/components/vertical-sections/index.js

```diff
@@ -1,15 +1,31 @@
 jQuery( document ).ready( function( $ ) {
+
+	// If the current screen doesn't have JS sections, return.
+	if ( 0 === $( '.edd-vertical-sections.use-js' ).length ) {
+		return;
+	}
+
 	// Hides the section content.
 	$( '.edd-vertical-sections.use-js .section-content' ).hide();
 
-	// Shows the first section's content.
-	$( '.edd-vertical-sections.use-js .section-content:first-child' ).show();
+	var hash = window.location.hash;
+	if ( hash && hash.includes( 'edd_' ) ) {
+		// Show the section content related to the URL.
+		$( '.edd-vertical-sections.use-js' ).find( hash ).show();
+
+		// Set the aria-selected for section titles to be false
+		$( '.edd-vertical-sections.use-js .section-title' ).attr( 'aria-selected', 'false' );
+
+		// Set aria-selected true on the related link.
+		$( '.edd-vertical-sections.use-js' ).find( '.section-title a[href="' + hash + '"]' ).parents( '.section-title' ).attr( 'aria-selected', 'true' );
```

Comment:
This `.section-title--is-active` class needs to be toggled alongside `aria-selected`.

Reply:
Thanks; I had missed it in another spot as well.